File: leetcode/4/exo.go
package main
import (
	"fmt"
	"sort"
)
func findMedianSortedArrays(nums1 []int, nums2 []int) float64 {
    var nums []int
    nums=append(nums1,nums2...)
	
	sort.Ints(nums)

	if len(nums)%2==0{
		return float64(nums[len(nums)/2-1]+nums[len(nums)/2])/2
	}else{
		return float64(nums[len(nums)/2])
	}

}
func main(){
	fmt.Println(findMedianSortedArrays([]int{1,2},[]int{3,4}))
}

File: leetcode/4/exo.py
from math import floor
from typing import List


class Solution:
  
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
       nums =nums1+nums2
       num=sorted(nums)
       mid = floor(len(num)/2)
       if len(nums)%2!=0:
          print(num[mid])
          return num[mid]
          
       else:
            print(num[mid-1]/2+num[mid]/2)
            return (num[mid-1]/2+num[mid]/2)  
solution=Solution()        
rezultat=solution.findMedianSortedArrays([1,2],[3,4])   
print(rezultat)       